### PR TITLE
ci: add changelog to image label when pushing new versions

### PR DESCRIPTION
**Pull request by jgrer**
- Changed release.yaml to gather the auto generated release notes into release_notes.md
- Added code to convert release_notes.md into a html file
- Passed this html file to the dockerfile for the extension using build args
- Set this html file in string format as the com.docker.extension.changelog parameter

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,7 @@
-IMAGE?=projectcopacetic/copacetic-docker-desktop-extension
+IMAGE?=jgrerr/copacetic-docker-desktop-extension
```
Comment (ashnamehrotra) [inline Makefile:1]:
```suggestion
IMAGE?=projectcopacetic/copacetic-docker-desktop-extension
```

**Comment by ashnamehrotra** [comment]
@jgrer can we also change the version in the install link in the readme on release? `docker extension install projectcopacetic/copacetic-docker-desktop-extension:0.1.0`

**Comment by jgrer** [comment]
> @jgrer can we also change the version in the install link in the readme on release? `docker extension install projectcopacetic/copacetic-docker-desktop-extension:0.1.0`

removed the example of the install command (not worth to update) and updated the link to point to the latest version of the extension

**Review by ashnamehrotra — APPROVED** [review]
LGTM

**Pull request merged**